dev/f5dc3e3e-ff41-45c9-93f2-86052e0178c5/device.lp

```diff
@@ -0,0 +1,19 @@
+(librepcb_device f5dc3e3e-ff41-45c9-93f2-86052e0178c5
+ (name "SGPC3")
+ (description "Ultra-low power gas sensor for indoor air quality applications.")
```

Comment:
Same here: Do you want to add a link to the datasheet in the description? https://www.sensirion.com/file/datasheet_sgpc3/.